server/@types/crm7/index.d.ts

```diff
@@ -14,6 +14,24 @@ type NumberRateAndCost = {
 }
 
 export interface Crm7Response extends CrmResponse {
+  mergedScheduleCostsData: {
+    schedule: ScheduleItem[]
+    officeUse: Totals
+    costTotals: CostTotals
+    timeTotals: TimeTotals
+    totals: Totals
+    lettersAndPhoneCalls: {
+      totals: { letters: NumberRateAndCost; telephoneCalls: NumberRateAndCost; total: number; solicitorCost: number }
```

Comment:
Remove semi-colons:-
```suggestion
      totals: {
        letters: NumberRateAndCost
        telephoneCalls: NumberRateAndCost
        total: number
        solicitorCost: number
      }
```